src/pages/home/report/ReportActionCompose.js

```diff
@@ -109,14 +108,32 @@ class ReportActionCompose extends React.Component {
     }
 
     /**
-     * Updates the visiblity state of the menu
+     * Updates the visibility state of the menu
      *
      * @param {Boolean} isMenuVisible
      */
     setMenuVisibility(isMenuVisible) {
         this.setState({isMenuVisible});
     }
 
+    /**
+     * Focus the composer text input
+     */
+    focus() {
+        if (this.textInput) {
+            this.textInput.focus();
+        }
+    }
+
+    /**
+     * Un-focus from the composer text input
+     */
+    blur() {
+        if (this.textInput) {
+            this.textInput.blur();
+        }
+    }
```

Comment:
This isn't being called. Is it supposed to be?

Reply:
Currently no. Initially I used to clear focus from the input before opening a modal, so that the keyboard doesn't get in the way. 
Since we decided to revert this logic is no longer used, but I left it there as a compliment to `focus`

Reply:
Thanks for explaining, would you mind removing it in that case. We'd prefer to exclude an unused function as we may never need it.

Reply:
Oki @Julesssss will do
Can I ask something for the future. It'll be easier to resolve such items if you say something like 
> This isn't being called. Is it supposed to be? We'd prefer to exclude an unused function as we may never need it.

So I know how to resolve this and skip some back and forth as I won't be able to address this today

Not that I'm in a hurry, but next week I'll be starting to work on other items, and I thought I was done with this one yesterday.